ci/pipeline.yml

```diff
@@ -62,14 +62,33 @@ jobs:
         args:
         - -exc
         - |
-          bosh ssh worker "sudo sh -c 'iptables -D INPUT -s 10.254.0.0/16 -d 169.254.0.2/32 -p udp -m udp --dport 53 -j ACCEPT || true'"
-          bosh ssh worker "sudo sh -c 'iptables -D INPUT -s 10.254.0.0/16 -d 169.254.0.2/32 -p tcp -m tcp --dport 53 -j ACCEPT || true'"
           bosh ssh iaas-worker "sudo sh -c 'iptables -D INPUT -s 10.254.0.0/16 -d 169.254.0.2/32 -p udp -m udp --dport 53 -j ACCEPT || true'"
           bosh ssh iaas-worker "sudo sh -c 'iptables -D INPUT -s 10.254.0.0/16 -d 169.254.0.2/32 -p tcp -m tcp --dport 53 -j ACCEPT || true'"
-          bosh ssh worker "sudo sh -c 'iptables -I INPUT 1 -s 10.254.0.0/16 -d 169.254.0.2/32 -p udp -m udp --dport 53 -j ACCEPT'"
-          bosh ssh worker "sudo sh -c 'iptables -I INPUT 1 -s 10.254.0.0/16 -d 169.254.0.2/32 -p tcp -m tcp --dport 53 -j ACCEPT'"
           bosh ssh iaas-worker "sudo sh -c 'iptables -I INPUT 1 -s 10.254.0.0/16 -d 169.254.0.2/32 -p udp -m udp --dport 53 -j ACCEPT'"
           bosh ssh iaas-worker "sudo sh -c 'iptables -I INPUT 1 -s 10.254.0.0/16 -d 169.254.0.2/32 -p tcp -m tcp --dport 53 -j ACCEPT'"
+  - task: iptables-worker-bosh-dns
+    tags: [iaas]
```

Comment:
Is this how the docker image knows where to build the container?

Reply:
And in this instance, that would be an `iaas-worker`

Reply:
Exactly. From our manifest to create these workers, you assign them a tag, then you can use that tag in the pipelines to assign jobs/tasks to that particular worker. 
https://github.com/18F/cg-deploy-concourse/blob/master/operations/iaas-worker.yml#L35